requirements.in

```diff
@@ -1,7 +1,7 @@
 babel~=2.0
 geopandas>=0.11.0
-howso-engine~=22.0
-howso-visuals~=1.0
+howso-engine~=22.1
```

Comment:
I don't think these versions actually need to be updated in this way? If my understanding of `~=` is correct, `~=22.0` should encompass anything `>=22.0.0` and so we only need the requirements to be re-generated.